pkgs/by-name/so/sonarr/package.nix

```diff
@@ -129,10 +150,11 @@ buildDotnetModule {
       # makes real HTTP requests
       "FullyQualifiedName!~NzbDrone.Core.Test.TvTests.RefreshEpisodeServiceFixture"
       "FullyQualifiedName!~NzbDrone.Core.Test.UpdateTests.UpdatePackageProviderFixture"
-
+    ] ++ lib.optionals stdenvNoCC.hostPlatform.isDarwin [
```

Comment:
```suggestion
    ] ++ lib.optionals stdenvNoCC.buildPlatform.isDarwin [
```

IIRC these tests fail when run on macOS, not when building for macOS.

Reply:
Good catch, fixed :)